pkg/resources/scim_integration_acceptance_test.go

```diff
@@ -213,10 +204,8 @@ func TestAcc_ScimIntegration_InvalidIncomplete(t *testing.T) {
 }
 
 func TestAcc_ScimIntegration_migrateFromVersion091(t *testing.T) {
```

Comment:
let's add one more test for migration (with setting enabled to false after and expecting changes) - it may be added in the next PR